mender-convert-package

```diff
@@ -202,6 +202,9 @@ disk_write_at_offset "${output_dir}/data.img" "${sdimg_path}" "${data_start}"
 log_info "Performing platform specific package operations (if any)"
 platform_package
 
+# Create bmap index
+run_and_log_cmd "/usr/bin/bmaptool create ${sdimg_path} > ${sdimg_path}.bmap"
```

Comment:
I would like to avoid having a default dependency on `bmap-tools` and would considered this a advanced feature that is not enabled by default.

For this reason I would like this to be a configuration option that is not enabled by default. Suggested for configuration name: 

`MENDER_IMAGE_BMAP=n`

While at it, please validate that `bmaptool` is installed and fail gracefully if tool is not found.

Reply:
Why?  The dependency is protected by the docker container so shouldn't be a problem.  The bmap file that is created can simply be ignored if people don't need/want it.

I will check for the tool since I suppose invoking the script outside of the container may be a use case but I'm hesitant to turn it off completely.

Reply:
Yes, it is protected with the docker container but would like to support the use-case without container with minimal dependencies. `bmap` is not something that users of `mender-convert` would be familiar with and do not want to add confusion.

> The bmap file that is created can simply be ignored if people don't need/want it.

It will be unnecessary created in majority of the cases wasting space/time (small but still) and another thing that can go wrong (chances are slim for this probably but still). I would really like to keep the default setup minima

Creating this file unconditionally will also create questions, what is this `sdimg.bmap` file, why is it created, how is it used? For this reason we would need to explain it, and unless we make the default approach to write SD card with `sdimg.bmap` I do not think this should on by default. It is quite easy to add for people know how to use it.